src/ui/include/EditorNS/editor.h

```diff
@@ -33,36 +36,32 @@ namespace EditorNS
     class Editor : public QWidget
     {
         Q_OBJECT
-    private:
-
     public:
-
-        struct Theme {
-            QString name;
-            QString path;
-        };
-
-        explicit Editor(const Theme &theme, QWidget *parent = 0);
         explicit Editor(QWidget *parent = 0);
 
         /**
-             * @brief Efficiently returns a new Editor object from an internal buffer.
-             * @return
-             */
+         * @brief Efficiently returns a new Editor object from an internal buffer.
+         * @return
+         */
         static QSharedPointer<Editor> getNewEditor(QWidget *parent = 0);
         static Editor *getNewEditorUnmanagedPtr(QWidget *parent);
-
         static void invalidateEditorBuffer();
 
+        //FIXME: Possibly un-necessary?
+        /*
         struct LanguageGreater {
```

Comment:
Do they actually need to be sorted?  From what I can tell QMap is sorting everything anyways... exactly like the sorting function was doing, lol.